server/utils.js

```diff
@@ -101,3 +103,17 @@ exports.stringTemplate = (configPath, ...args) => {
 
   return ''
 }
+
+const readFileAsync = promisify(fs.readFile)
+exports.readFileAsync = readFileAsync
+exports.assetDataURI = async (filePath) => {
+  // If the path starts with `/assets`, look in the app’s public directory
+  const publicPath = filePath.replace(/^\/assets/, '/public')
+
+  const mimeType = mime.lookup(path.posix.basename(publicPath))
```

Comment:
Good questions! My understanding of [the node docs](https://nodejs.org/api/path.html#path_windows_vs_posix) is that because we're using URL-style (and thus POSIX-style) paths here, we need to call `path.posix.basename` to get consistent behavior in both OS families.

> To achieve consistent results when working with POSIX file paths on any operating system, use path.posix

But I'm not set up to test this on Windows, and I may well be misreading the docs. Let me know if you'd like to drop to just `path.basename`.

Reply:
Ah, that makes sense. I think you are reading the docs correctly and we should leave this line with the use of `path.posix` as you have done so far. 

Would it make sense to add a 1 line comment explaining the rationale, since it might also strike others as a potential source of bugs? (We don't specify `path.posix` anywhere else in the codebase, which is why this one jumped out at me)

Reply:
Good idea — will add that comment.